zincware/ZnTrack
# add config.log_level

# changes
* run all notebooks with the reduced logging output
* fix the reloading of data in project if it worked
* fix live output for running dvc commands

# How to test
look through the notebooks and see if the amount of produced outputs looks good. Is there anything import missing? Is there still too much information?
